index.bs

```diff
@@ -60,12 +60,12 @@ specification uses that specification and terminology.
 
 # Terminology #  {#terminology}
 
-A chunk is a piece of data. In the case of CompressionStream and DecompressionStream, the chunk type is Uint8Array. 
+A chunk is a piece of data. In the case of CompressionStream and DecompressionStream, the output chunk type is Uint8Array. They accepts any {{BufferSource}} type as input.
 
 A stream represents an ordered sequence of chunks. The terms {{ReadableStream}} and {{WritableStream}} are defined in
 [[!WHATWG-STREAMS]].
 
-Deflate is a compression format. Gzip is another compression format, also based on the deflate algorithm. Both deflate and gzip are defined in zlib.
+Deflate is a compression format defined in [[!RFC1950]]. It is referred to there as "ZLIB", but in this standard we call it "deflate" to match HTTP (see [[RFC7230]] section 4.2.2). Gzip is another compression format defined in [[!RFC1952]], also based on the deflate algorithm.
```

Comment:
I think this needs to be be `[[!RFC7230]]`, not `[[RFC7230]]` (I haven't checked that the link actually works).

Reply:
I was wrong, sorry. `[[RFC7230]] ` actually does the right thing, by creating an "Informative reference". So please don't change this part.